docs/proposals/perf.md

```diff
@@ -0,0 +1,364 @@
+
+When using the KubeEdge Performance Test, the developer is responsible for creating #1 and #2 above.
+
+Test Client will do the rest of the setup like building KubeEdge EdgeNode image and push the image to Docker repository.
+Once the image is pushed sucessfully, Test Client uses the deployment object to deploy KubeEdge Nodes as pods
+and waits until all the pods come up and **Running**.
+
+#### K8S Master
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| K8S Version                    |  v1.13.5                                     |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1                                           |
+
+This VM is used to run K8S Master Services including K8S API Server and K8S Scheduler and so on.
+
+#### KubeEdge Cloud Part
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1                                           |
+
+This VM is used to run KubeEdge Cloud Part Services including Edge Controller and Cloud Hub and so on.
+
+#### K8S Nodes
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1...N                                       |
+
+These VMs are used to deploy numbers of KubeEdge Edge Nodes pods which are running Edged and EdgeHub and so on.
+We will adjust the Count of VMs based on the KubeEdge Edge Nodes numbers.
+
+KubeEdge Performance Test setup is similar with K8S KubeMark setup,
```

Comment:
The KubeMark like deployment is for simulating the control-plane scalability.
You may also describe the simple setup that used for testing edgenode CPU/Mem usage etc.

Reply:
The deployment figure is updated. Thanks